src/PKSim.Presentation/Services/OntogenyTask.cs

```diff
@@ -62,7 +62,7 @@ public ICommand LoadOntogenyForMolecule(IndividualMolecule molecule, TSimulation
          };
          dataImporterSettings.AddNamingPatternMetaData(Constants.FILE);
 
-         var data = _dataImporter.ImportDataSets(new List<MetaDataCategory>(), getColumnInfos(), dataImporterSettings).DataRepositories.FirstOrDefault();
+         var data = _dataImporter.ImportDataSets(new List<MetaDataCategory>(), getColumnInfos(), dataImporterSettings).DataRepositories?.FirstOrDefault();
```

Comment:
plus DataRepositories SHOULD NEVER BE NULL. I think I made a PR for this so this should never happen again
@georgeDaskalakis @abdelr Do you concur?

Reply:
I do not think there is a situation when DataRepositories are null now. Plus, this has not been merged and I do not see the crash anymore (after fixing the configuration issue). Could we wait to merge this? I am not sure we need this check. Actually, the error listed on [1790](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1790) refers to the same in [Core/1099](https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/pull/1099). I would reject this PR and close [1790](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1790) in favor of [Core/1099](https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/pull/1099) since the problem is solved there.

Reply:
agreed